### PR TITLE
Add new controlsMappings inline with SOC2

### PR DESCRIPTION
**Pull request by henrymgarrett**

### CODE REVIEW

File: templates/standards/soc2-security.json [hunk 1]
```diff
@@ -10,27 +10,62 @@
...
+            "cp-sanctions",
+            "cp-training-policy"
+          ]
         },
         {
           "ref": "CC1.2",
           "title": "COSO Principle 2",
-          "summary": "The board of directors demonstrates independence from management and exercises oversight of the development and performance of internal control."
+          "summary": "The board of directors demonstrates independence from management and exercises oversight of the development and performance of internal control.",
+          "mappedControls": []
```
Comment (erkangz) [inline templates/standards/soc2-security.json:27]:
```suggestion
          "mappedControls": ["cp-gov-bod"]
```

**Review by erkangz — CHANGES_REQUESTED** [review]

File: templates/standards/soc2-security.json (hunk 1 above)
Comment (henrymgarrett) [reply templates/standards/soc2-security.json:27]: I can make that happen

**Review by erkangz — APPROVED** [review]

**Comment by erkangz** [comment]
@henrymgarrett how about we move this file into templates/standards/soc2/soc2-security.json (adding `soc2` folder)?

We can then add the other appropriate ones for confidentiality, availability, integrity, and privacy into the folder next.

**Comment by henrymgarrett** [comment]
> @henrymgarrett how about we move this file into templates/standards/soc2/soc2-security.json (adding `soc2` folder)?
> 
> We can then add the other appropriate ones for confidentiality, availability, integrity, and privacy into the folder next.

That's going to break the v1 service, it uses these URLs directly (which I dislike).  We will need to make code changes first before doing that, but I do agree, that will be nicer.  How about if I merge this, and then we can do that in a follow up PR? @erkangz

**Comment by erkangz** [comment]
> > @henrymgarrett how about we move this file into templates/standards/soc2/soc2-security.json (adding `soc2` folder)?
> 
> > 
> 
> > We can then add the other appropriate ones for confidentiality, availability, integrity, and privacy into the folder next.
> 
> 
> 
> That's going to break the v1 service, it uses these URLs directly (which I dislike).  We will need to make code changes first before doing that, but I do agree, that will be nicer.  How about if I merge this, and then we can do that in a follow up PR? @erkangz 

What if we keep them here and copy it to `soc2` directory so that v2 can start using it from there? We can then remove the duplicate when we fully roll out v2 and decommission v1.

**Comment by henrymgarrett** [comment]
> What if we keep them here and copy it to `soc2` directory so that v2 can start using it from there? We can then remove the duplicate when we fully roll out v2 and decommission v1.

Great idea 👐🏼

**Review by erkangz — APPROVED** [review]

**Pull request merged**